.github/workflows/sync-integration-templates.yaml

```diff
@@ -0,0 +1,39 @@
+name: 'Sync Integration Templates'
```

Comment:
I am not sure I follow why we need this extra action. Aren't the merge triggering the templates upload action above done by us, even on external contributions? if yes what's the permissions issue?

Reply:
It is more of a fail safe than anything. It’s a force sync of all templates since the regular one looks just for changed files. 

For some reason this action contribution https://github.com/NangoHQ/nango/pull/2242 didnt get uploaded to s3. In that instance we need a way to sync integration templates. I need to look into exactly why it failed but I assumed it was a permissions issue but not 100% sure.

Reply:
suggestion for future: this could be done at the start of the server process instead